<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorWizard</class>
 <widget class="QFrame" name="ColorWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_0">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="add_button">
        <property name="toolTip">
         <string>Add to favorites</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../res/extui/extui.qrc">
          <normaloff>:/extui/char_pm.png</normaloff>:/extui/char_pm.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ColorComboBox" name="combo">
        <property name="toolTip">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="colorwin_button">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../res/ui/ui.qrc">
          <normaloff>:/ui/colorxx.png</normaloff>:/ui/colorxx.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="fav_list"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>colorcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../res/extui/extui.qrc"/>
  <include location="../../../res/ui/ui.qrc"/>
 </resources>
 <connections/>
</ui>
